services/tenant-ui/frontend/src/components/profile/Developer.vue

```diff
@@ -2,14 +2,67 @@
   <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
   <h2>{{ $t('profile.config') }}</h2>
   <vue-json-pretty :data="config" />
+  <h2>{{ $t('profile.token') }}</h2>
```

Comment:
Good practice moving forward is to maybe encapsulate these into their own components